ReactAndroid/src/main/java/com/facebook/react/ReactActivity.java

```diff
@@ -223,4 +223,13 @@ public void onBackPressed() {
   public void invokeDefaultOnBackPressed() {
     super.onBackPressed();
   }
+
+  @Override
+  public void onNewIntent(Intent intent) {
```

Comment:
Can you add Javadoc explaining why this is needed? When someone sees this code later they won't see the description of this pull request anymore.